gradio/processing_utils.py

```diff
@@ -341,6 +342,24 @@ def move_resource_to_block_cache(
     return block.move_resource_to_block_cache(url_or_file_path)
 
 
+def is_file_obj(data: dict | JsonData):
```

Comment:
Is this method only added for type-checking purposes? Because `client_utils.is_file_obj` already checks to see if `data` is a `dict`?

Reply:
Yea you're right another brain fart hehe